cpp/src/search/search.cu

```diff
@@ -255,33 +267,51 @@ struct multi_contains_dispatch {
 };
 
 template <>
-std::unique_ptr<column> multi_contains_dispatch::operator()<dictionary32>(
+std::unique_ptr<column> multi_contains_dispatch::operator()<list_view>(
   column_view const& haystack,
   column_view const& needles,
   rmm::mr::device_memory_resource* mr,
   cudaStream_t stream)
 {
-  CUDF_FAIL("dictionary type not supported");
+  CUDF_FAIL("list_view type not supported");
 }
 
 template <>
-std::unique_ptr<column> multi_contains_dispatch::operator()<list_view>(
+std::unique_ptr<column> multi_contains_dispatch::operator()<struct_view>(
   column_view const& haystack,
   column_view const& needles,
   rmm::mr::device_memory_resource* mr,
   cudaStream_t stream)
 {
-  CUDF_FAIL("list_view type not supported");
+  CUDF_FAIL("struct_view type not supported");
 }
 
 template <>
-std::unique_ptr<column> multi_contains_dispatch::operator()<struct_view>(
-  column_view const& haystack,
-  column_view const& needles,
+std::unique_ptr<column> multi_contains_dispatch::operator()<dictionary32>(
+  column_view const& haystack_in,
+  column_view const& needles_in,
```

Comment:
why is `_in` appended?

Reply:
Why not? These names are only used to convert them to `dictionary_column_view` instances. It is less typing and easier to follow in my opinion.